src/einsteinpy/symbolic/tensor.py

```diff
@@ -121,6 +122,24 @@ def config(self):
     def __getitem__(self, index):
         return self.arr[index]
 
+    def __str__(self):
+        """
+        Returns a String with a readable representation of the object of class Tensor
+
+        """
+        representation = "Tensor" + "\n"
+        representation += self.arr.__str__()
```

Comment:
Reomve this change, I guess this is handled by #279

Reply:
Remove each and every change in `tensor.py`

Reply:
Yes, they are the #279 PR's modifications. I remove them